src/test/java/uk/gov/pay/ledger/pact/ContractTest.java

```diff
@@ -346,6 +349,30 @@ public void createAPaymentWithAllFieldsAndWithACorrespondingRefund(Map<String, S
 
     }
 
+    @State("two payouts exist for selfservice search")
+    public void createTwoPayouts() {
+        String gatewayAccountId = "654321";
+        aPayoutFixture()
+                .withGatewayAccountId(gatewayAccountId)
+                .withGatewayPayoutId("payout-id-1")
+                .withAmount(1250L)
+                .withCreatedDate(ZonedDateTime.parse("2020-05-21T12:22:16.067Z"))
+                .withPaidOutDate(ZonedDateTime.parse("2020-05-22T14:22:16.067Z"))
+                .withState(PayoutState.PAID_OUT)
+                .build()
+                .insert(app.getJdbi());
+
+        aPayoutFixture()
+                .withGatewayAccountId(gatewayAccountId)
+                .withGatewayPayoutId("payout-id-2")
+                .withAmount(2345L)
+                .withCreatedDate(ZonedDateTime.parse("2020-05-22T12:22:16.067Z"))
```

Comment:
I think consumer tests could become flakey as created_date is same for 2 payout records and possibly down to DB which order records are returned ?

Reply:
Not the same, one is `2020-05-21T12:22:16.067Z` and the other one is `2020-05-22T12:22:16.067Z`

Reply:
ah, I missed date part